packages/text-annotator-react/test/App.tsx

```diff
@@ -22,13 +22,6 @@ const TestPopup = (props: TextAnnotatorPopupProps) => {
     anno.cancelSelected();
   };
 
-  useEffect(() => {
-    const { current: inputEl } = inputRef;
-    if (!inputEl) return;
-
-    setTimeout(() => inputEl.focus({ preventScroll: true }));
-  }, []);
-
```

Comment:
It's better for accessibility to not shift the focus to the popup content right after/during the selection. Here's the a11y expert comment on that matter:

> Ideally, you could release shift, then press it again, and select some more text. This is true also for Mouse users in order to pass a new SC in WCAG 2.2: https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements This is for people who have trouble dragging. This makes tracking Shift (rather than something admittedly awkward like Control Alt/Option h) tricky. But if you could leave focus on the page, and just open the popup when mouse is released or when shift is lifted, and then leave it open through subsequent shift presses and shift clicks, and let users tab to the popup or click a close button or press escape to close the popup, that would work